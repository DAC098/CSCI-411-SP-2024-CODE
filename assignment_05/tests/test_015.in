16 904
100
1964
2299
2867
4007
4008
4421
4629
5376
7012
7033
7605
7997
8710
9718
9766
