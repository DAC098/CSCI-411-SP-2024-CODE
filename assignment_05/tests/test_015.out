Enter the number of hotels and the ideal number of miles to travel per day: Enter 16 hotel distances each on a separate line: Hotels to visit: 2 4 5 8 9 10 13 14 16 
Total penalty: 1885440
